### PR TITLE
[JENKINS-61798] Fix launch issue

### PR DESCRIPTION
**Pull request by slide**
Set the property of the launch right before it is launched so that the correct port is used.

cc @oleg-nenashev

### CODE REVIEW

**Comment by daniel-beck** [comment]
> NOT Installed AND NOT REMOVE

Ungrammatical?

**Comment by slide** [comment]
Yes. welcome to the wonderful world of Windows Installers.

File: msi/build/jenkins.wxs [hunk 1]
```diff
@@ -460,12 +460,12 @@
       <Publish Dialog="MaintenanceTypeDlg" Control="RemoveButton" Event="NewDialog" Value="VerifyReadyDlg">1</Publish>
       <Publish Dialog="MaintenanceTypeDlg" Control="Back" Event="NewDialog" Value="MaintenanceWelcomeDlg">1</Publish>
 
-      <Publish Dialog="ExitDialog" Control="Finish" Property="WixShellExecTarget" Value="http://localhost:[PORTNUMBER]/" Order="1">NOT Installed</Publish>
-      <Publish Dialog="ExitDialog" Control="Finish" Event="DoAction" Value="LaunchApplication" Order="2">NOT Installed</Publish>
+      <Publish Dialog="ExitDialog" Control="Finish" Property="WixShellExecTarget" Value="http://localhost:[PORTNUMBER]/" Order="1">NOT Installed AND NOT REMOVE</Publish>
```
Comment (olblak) [inline msi/build/jenkins.wxs:463]:
```suggestion
      <Publish Dialog="ExitDialog" Control="Finish" Property="WixShellExecTarget" Value="http://localhost:[PORTNUMBER]/" Order="1">NOT Installed AND NOT REMOVED</Publish>
```
Why some REMOVED capitalized and not Installed?

File: msi/build/jenkins.wxs [hunk 2]
```diff
@@ -460,12 +460,12 @@
...
+      <Publish Dialog="ExitDialog" Control="Finish" Event="DoAction" Value="LaunchApplication" Order="2">NOT Installed AND NOT REMOVE</Publish>
```
Comment (olblak) [inline msi/build/jenkins.wxs:464]:
```suggestion
      <Publish Dialog="ExitDialog" Control="Finish" Event="DoAction" Value="LaunchApplication" Order="2">NOT Installed AND NOT REMOVED</Publish>
```

File: msi/build/jenkins.wxs (hunk 1 above)
Comment (olblak) [reply msi/build/jenkins.wxs:463]: for some reason, I didn't see that @daniel-beck  already rised the same concern than me, so I remove my suggestion

**Pull request merged**